tests/unit/bottomedge/tst_bottomedge.cpp

```diff
@@ -855,15 +855,14 @@ private Q_SLOTS:
     {
         QFETCH(QString, document);
 
-        QScopedPointer<BottomEdgeTestCase> test(new BottomEdgeTestCase(document));
+        auto test = new BottomEdgeTestCase(document);
```

Comment:
Has Qt's memory handling somehow improved to make the ScopedPointer no longer necessary, or is there another reason to use auto instead?

Reply:
I'm not sure why the ScopedPointer is used in these tests exactly, as I don't think it should be necessary. These changes were made due to the test crashing. However, reverting this seems to work again (I am not sure why). So assuming the new build succeeds in CI, I think we can consider this resolved too.